assets/js/googlesitekit/modules/datastore/sharing-settings.test.js

```diff
@@ -612,5 +616,143 @@ describe( 'core/modules sharing-settings', () => {
 				).toBe( false );
 			} );
 		} );
+
+		describe( 'haveSharingSettingsExpanded', () => {
+			it( 'requires the `key` param', () => {
+				expect( () => {
+					registry
+						.select( CORE_MODULES )
+						.haveSharingSettingsExpanded( 'invalid-param' );
+				} ).toThrow( 'key must be one of: management, sharedRoles.' );
+			} );
+
+			it( 'should return undefined if `sharingSettings` or `savedSharingSettings` cannot be loaded', () => {
+				global[ dashboardSharingDataBaseVar ] = undefined;
+
+				expect(
+					registry
+						.select( CORE_MODULES )
+						.haveSharingSettingsExpanded( 'management' )
+				).toBeUndefined();
+			} );
+
+			it( 'informs whether the `management` setting for any module has been changed from `owner` to `all_admins`', async () => {
```

Comment:
With no use of `await` in the test, this `async` is redundant

```suggestion
			it( 'informs whether the `management` setting for any module has been changed from `owner` to `all_admins`', () => {
```